pandas/core/generic.py

```diff
@@ -10596,11 +10607,12 @@ def _doc_parms(cls):
 DataFrame.all : Return whether all elements are True over requested axis.
 """
 
-_any_desc = """\
-Return whether any element is True over requested axis.
+_any_doc = """\
```

Comment:
I guess you changed the name to be consitent with the `all`, but it should be the other way round, this should be named `_any_summary` (or `_any_desc`). For the convention we use `_any_doc` should be the whole docstring of the `any` method.

Reply:
Yes. I did wonder if that was the right way round. I'll change them both to `_desc`.

I also considered moving the `any` variables to be close to the `all` and `_bool_doc` ones. What do you think?